### DISCUSSION TITLE
Add `Tag` To Linked Anim Graphs in `AB_Als`

### DISCUSSION THREAD

**Discussion by Benbentwo** [Sixze/ALS-Refactored#427] [Ideas]
Adding `Tag`s to our  Linked animation graph allows them to be easily overridden by the function `Link Anim Graph by Tag` Node. This allows customization without forking for projects. We can then create Child ABP's of things like `AB_Als_Standing` and link them by tag to override custom movement ABPs

![image](https://github.com/Sixze/ALS-Refactored/assets/14911525/160433fd-378d-44e8-a875-ae281d4b4ce7)

**Comment by Sixze** [comment]
Done in 38c966c3f78a1973a1edaa657ab1185f3df982b1.
